modular_skyrat/modules/opposing_force/code/opposing_force_equipment.dm

```diff
@@ -0,0 +1,572 @@
+/**
+ * OPFOR EQUIPMENT DATUM
+ *
+ * The OPFOR subsystem will gather all of these on initialisation and populate a list, which is then passed
+ * into the OPFOR UI, which can be selected by the user.
+ *
+ * The User will be equipped with whatever they have selected if the approving admin authorises it.
+ */
+
+
```

Comment:
This is documentation, no need to make it separate. If you could beef it up just a bit, just to make sure to convey what the use of this datum is.
```suggestion
/**
 * OPFOR EQUIPMENT DATUM
 *
 * The OPFOR subsystem will gather all of these on initialisation and populate a list, which is then passed
 * into the OPFOR UI, which can be selected by the user.
 *
 * The User will be equipped with whatever they have selected if the approving admin authorises it.
 */
```